pep-0XXX-environment-markers.rst

```diff
@@ -0,0 +1,171 @@
+PEP: 0XXX
+Title: Environment Markers
+Version: $Revision$
+Last-Modified: $Date$
+Author: James Polley <jp@jamezpolley.com>
+BDFL-Delegate: Nick Coghlan <ncoghlan@gmail.com>
+Status: Draft
+Type: Informational
+Content-Type: text/x-rst
+Created: 03-Jul-2015
+Post-History: 03-Jul-2015
+
+
+Abstract
+========
+
```

Comment:
I think this starts too deep. If it started with something like the below it might be nice.

Packages need to have dependencies that are conditional on the environment the package is being installed into. Environment markers provide a small language for describing whether or not a particular dependency will apply to a given environment.